release-index.yaml

```diff
@@ -5,6 +5,14 @@ releases:
         version: d2657773d007e1b4c0b2dd715c628d24c0d7b5fb
       - name: revert-ubuntu-22-04
         version: 1ff0e709f0d0984a4f9ab06456db177c4b6e48a0
+      - # This has three reverts and rides atop the base release
+        # Here is the successful test: https://github.com/dfinity/ic/actions/runs/11241522710
+        name: canister-overhead-hotfix
+        version: f0c923eba09e9c1444501692b0ab4884882bf5bc
+      - # This has the same reverts as the prior but rides atop the revert-ubuntu-22-04 release
+        # Here is the test battery: https://github.com/dfinity/ic/actions/runs/11243607016
+        name: revert-ubuntu-22-04-canister-overhead-hotfix
+        version: 7c50c0db1e247a25896d77da72c410ff17ef5773
```

Comment:
This wasn't really needed. This whole version is just a backup if base (which has the last weeks feature) went south because of said feature. But its okay.

Reply:
Excellent.  I have commented it.